Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/05-PersistingSyncUps/PersistingSyncUps-02-code-0001-previous.swift

```diff
@@ -39,6 +39,21 @@ struct SyncUpsListTests {
 
   @Test
   func deletion() async {
-    // ...
```

Comment:
📓 With the addition of withLock, we need to fix some compilation errors in the tests, so I had to had new steps to explain what need to be done before being able to say that everything still passes as before like it was originally.